suites/apis/googleDataAccessTest.js

```diff
@@ -471,11 +471,11 @@ Scenario('Test Google Data Access (signed urls and temp creds) @reqGoogle @googl
   //  - Check signed URLs from FIRST RUN
   chai.expect(User0AccessRemovedQA,
     "Make sure signed URL from User0 CANNOT access QA data again. FAILED."
-  ).to.not.equal(fence.props.googleBucketInfo.QA.fileContents);
+  ).to.contain('AccessDenied');
 
   chai.expect(User1AccessRemovedQA,
     "Make sure signed URL from User1 CANNOT access QA data again. FAILED."
-  ).to.not.equal(fence.props.googleBucketInfo.QA.fileContents);
+  ).to.contain('AccessDenied');
```

Comment:
where does `AccessDenied` come from? shouldn't it return `GOOGLE_FILE_FROM_URL_ERROR`?

Reply:
So turns out google will still allow accessing the signed url but they send back `AccessDenied` in the response